plugin.json

```diff
@@ -1,28 +1,30 @@
-{
-    "label": {
-        "de": "dotdigital Engagement Cloud",
-        "en": "dotdigital Engagement Cloud"
-    },
-    "copyright": "Copyright (c) 2017 dotDigital Group PLC",
-    "license": "MIT",
-    "link": "https://www.dotdigital.com/",
-    "author": "dotdigital",
-
-    "currentVersion": "1.0.0",
-
-    "changelog": {
-        "de": {
-            "1.0.0": "Erstveröffentlichung"
-        },
-        "en": {
-            "1.0.0": "First release"
-        }
-    },
-
-    "compatibility": {
-        "minimumVersion": "4.3.0",
-        "maximumVersion": null,
-        "blacklist": [
-        ]
-    }
-}
+{
+    "label": {
+        "de": "dotdigital Engagement Cloud",
+        "en": "dotdigital Engagement Cloud"
+    },
+    "copyright": "Copyright (c) 2017 dotdigital Group PLC",
+    "license": "MIT",
+    "link": "https://www.dotdigital.com/",
+    "author": "dotdigital",
+
+    "currentVersion": "1.0.0",

```

Comment:
Should this be 1.0.1? Is that why the plugin manage displays 1.0.0?

Reply:
Might be, I must missed this one when I fixed the file!